docs/utils.md

```diff
@@ -0,0 +1,26 @@
+# Utility Subcommands
+
+Besides the Live-Trade and Dry-Run run modes, the `backtesting`, `edge` and `hyperopt` optimization subcommands, and the `download-data` subcommand which prepares historical data, the bot contains a number of utility subcommands. They are described in this section.
+
+## List Exchanges
+
+```
```

Comment:
I would add a sentense to each of the commands that quickly explains what it's usefull for.

kinda like
`Use this to find out which exchanges are supported by freqtrade`

and 
`Use this to see which timeframes are available for your exchange` ... (or similar ...)

Reply:
Done, also added some examples